lessons/Data-Science/box-plots/langs/en-us/pages/match-dot-plots-summaries.adoc

```diff
@@ -0,0 +1,87 @@
+= Matching Dot Plots and Five-Number Summaries
+
+++++
+<style>
+/* Format matching answers to render with an arrow */
+.solution::before{ content: ' → '; }
+</style>
+++++
+Draw a line from each dot plot on the left to the corresponding five-number summary on the right.
+
+[.FillVerticalSpace, cols="^.^15a,^.^3a,5a,^.^1a,<.^5a", options="header", stripes="none", grid="none", frame="none"]
+|===
+| Dot Plot
+|||
+| Five-Number Summary
+
+| @image{../images/dp1.png }
+|*@n* @ifsoln{C}  ||*A*
+| Min: 2 +
+Quartile 1: 4 +
```

Comment:
I'm finding this hard to read with the colon between two digits

> Quartile 1: 4

Guessing you thought a lot about the choice. Preference for replacing it with Q1 or First Quartile?

Reply:
happy to go with Q1. Will change.